pkgs/development/tools/misc/universal-ctags/default.nix

```diff
@@ -12,7 +12,8 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ pythonPackages.docutils ];
-  buildInputs = [ autoreconfHook pkgconfig ];
+  buildInputs = [ autoreconfHook pkgconfig ] ++
+    (stdenv.lib.optional stdenv.isDarwin [ libiconv ]);
```

Comment:
this results in a nested list, use `optional stdenv.isDarwin libiconv` or `optionals stdenv.isDarwin [ libiconv ... ]`

Reply:
Ugh, thank you!